core/src/main/java/org/mapstruct/BeanMapping.java

```diff
@@ -18,7 +18,9 @@
 /**
  * Configures the mapping between two bean types.
```

Comment:
Forces MapStruct to handle target and source as bean and configures the mapping.